perf/src/sigverify.rs

```diff
@@ -551,16 +567,24 @@ pub fn ed25519_verify(
         num_packets = num_packets.saturating_add(batch.packets.len());
     }
     out.resize(signature_offsets.len(), 0);
-    trace!("Starting verify num packets: {}", num_packets);
-    trace!("elem len: {}", elems.len() as u32);
-    trace!("packet sizeof: {}", size_of::<Packet>() as u32);
+    warn!(
+        "Starting verify {}, num packets: {}",
+        P::get_packet_type_name(),
+        num_packets
+    );
+    warn!("elem len: {}", elems.len() as u32);
+    warn!("packet sizeof: {}", size_of::<P>() as u32);
+    // todo: figure out what this means; the data field max size
```

Comment:
what are you trying to figure out? `PACKET_DATA_SIZE` should be the same as `size_of::<Packet>()` so that's what the trace log was probably for

Reply:
I was mostly just wondering what this information this debug message was trying to convey, and if it had any importance now that we're modifying the max data size in the ExtendedPacket case. It was kind of confusing since the max data size doesn't actually seem to be used in figuring out any of the offsets. Not a huge deal, and will remove the TODO soon.